sysl2/sysl/cmd_codegen.go

```diff
@@ -21,6 +21,7 @@ func (p *codegenCmd) Configure(app *kingpin.Application) *kingpin.CmdClause {
 		Default(".").StringVar(&p.rootTransform)
 	cmd.Flag("transform", "grammar.g").Required().StringVar(&p.transform)
 	cmd.Flag("grammar", "grammar.g").Required().StringVar(&p.grammar)
+	cmd.Flag("app", "name of the app").Required().StringVar(&p.appName)
```

Comment:
please use a more descriptive flag name. Also does this flag makes sense on any other commands?

Reply:
I think the help string could be a little more descriptive.

Reply:
Nope. It's only required for codegen.

Reply:
Updated description